maximize-select2-height.js

```diff
@@ -65,8 +65,9 @@
       // ------------------------------------------
       //   total height available to us
 
-      // innerHeight is more accurate across browsers than $(window).height().
-      height = window.innerHeight +
+      // innerHeight is more accurate across browsers than $(window).height(),
+      // but clientHeight works better than innerHeight on mobile browsers.
+      height = window.document.documentElement.clientHeight +
```

Comment:
This is based on this suggestion: https://github.com/panorama-ed/maximize-select2-height/issues/13#issuecomment-409389595

I haven't tested this on a mobile browser, but I've tested (using the `test.html` file in this repo) this on Safari and Chrome and it appears to still work with this change. I tried using Chrome's mobile browser mode in devtools but it doesn't show the address bar so I can't verify that this fixes the issue described in #13. But I don't see a harm in making this change and I'm willing to assume the user is correct, and based on some Googling about `window.innerHeight` vs. `window.document.documentElement.clientHeight` the change makes sense.